server/views/templates/article.njk

```diff
@@ -3,31 +3,33 @@
 {% block articleBody %}
   <div class="body-content">
     {% for bodyPart in bodyParts %}
-      <div class="grid grid--dividers grid--flush-dividers">
-      {% if bodyPart.copy %}
-        <div class="grid__cell grid__cell--xl6 grid__cell--shift-xl1 grid__cell--l7 grid__cell--m6 grid__cell--shift-m1 grid__cell--s4">
-          {{ bodyPart.copy | safe }}
-        </div>
-        {% if bodyPart.figure %}
-          <div class="grid__cell grid__cell--xl4 grid__cell--l5 grid__cell--m6 grid__cell--shift-m1 grid__cell--s4">
+      <div class="container">
+        <div class="grid grid--dividers grid--flush-dividers">
+        {% if bodyPart.copy %}
+          <div class="grid__cell grid__cell--xl6 grid__cell--shift-xl1 grid__cell--l7 grid__cell--m6 grid__cell--shift-m1 grid__cell--s4">
+            {{ bodyPart.copy | safe }}
+          </div>
+          {% if bodyPart.figure %}
+            <div class="grid__cell grid__cell--xl4 grid__cell--l5 grid__cell--m6 grid__cell--shift-m1 grid__cell--s4">
+              {% component 'figure', {
+                type: 'picture',
+                sources: bodyPart.figure.sources,
+                caption: bodyPart.figure.caption,
+                modifiers: ['sticky'],
+                icon: bodyPart.figure.icon } %}
+            </div>
+          {% endif %}
+        {% elif bodyPart.figure %}
+          <div class="grid__cell grid__cell--l12 grid__cell--m8, grid__cell--s4">
```

Comment:
This is a change in order to keep up with the changes that have already occurred in the article page. Not ideal, obviously.